doc/project/project_roles.rst

```diff
@@ -314,15 +314,37 @@ Release Activity
 Merge Criteria
 ++++++++++++++
 
-* All continuous integration checks have passed
+* Minimal of 2 approvals:
+
+  * A collaborator from the same subsystem.
+  * Alternately another maintainer of another subsystem
+  * Approval by the assignee
+
+* Four eye principle on the organisation level.
+
+  * Changes or additions to common and shared code shall have reviews from
+    different organisations (at least one review and approval from a different
+    submitters' organisation).
+  * Changes or additions to hardware support (driver, SoC, boards) shall at
```

Comment:
Does this refer to common driver APIs? or to any driver, soc, or board related code?
(because if the 2nd, I do not expect people from other organizations to prioritize the time to review and understand code that is specific to somebody else's boards or drivers)

Reply:
no, driver, soc and board implementations, not APIs. addressed.